backend/api/Database/Context/InitDb.cs

```diff
@@ -134,6 +134,9 @@ private static List<Area> GetAreas()
             MapMetadata = new MapMetadata(),
             DefaultLocalizationPose = new Pose { },
             SafePositions = new List<SafePosition>()
+            {
```

Comment:
Why is this not needed for the other areas?

Reply:
Is it needed?

Reply:
Yeah, ideally we need the areas to have a safe zone for the functionality to work. This area is the one we have been using to test the functionality so we added a safe zone in the initDb to make the process easier